src/panels/config/automation/trigger/ha-automation-trigger-row.ts

```diff
@@ -110,7 +110,7 @@ export default class HaAutomationTriggerRow extends LitElement {
 
         <ha-expansion-panel
           leftChevron
-          .header=${describeTrigger(this.trigger)}
+          .header=${this.trigger.alias || describeTrigger(this.trigger)}
```

Comment:
shouldnt `describeTrigger` us the alias if available?

Reply:
You are right, will adjust!